index.html

```diff
@@ -2012,8 +2012,8 @@ <h2>Data Schema Vocabulary Definitions</h2>
           <p>
             In a TD, concrete data formats are specified in Forms (see <a href="#form"></a>) using content types. When
             the value of a content type in an instance of the Form is <code>application/json</code>, the data schema can
-            be processed directly by JSON Schema processors. Otherwise, Web of Things (WoT) Binding Templates
-            [[?WOT-BINDING-TEMPLATES]] defines data schema's available mappings to other content types such as XML
+            be processed directly by JSON Schema processors. Otherwise, entries in Web of Things (WoT) Bindings Registry
+            [[?WOT-BINDINGS-REGISTRY]] defines data schema's available mappings to other content types such as XML
```

Comment:
```suggestion
            be processed directly by JSON Schema processors. Otherwise, entries in the Web of Things (WoT) Bindings Registry
            [[?WOT-BINDINGS-REGISTRY]] define available mappings from data schemas to other content types such as XML
```

Reply:
added to template file